src/Compilers/Core/Portable/ConstantValueSpecialized.cs

```diff
@@ -660,6 +692,22 @@ public override uint UInt32Value
                 }
             }
 
+            public override long Int64Value
+            {
+                get
+                {
+                    switch (this.Discriminator)
```

Comment:
This method appears in the specialized class whose purpose is to implement precisely these two discriminator kinds.